src/test/java/org/apache/maven/model/jdom/it/DependenciesEtlIT.java

```diff
@@ -29,6 +29,13 @@
  */
 public class DependenciesEtlIT extends AbstractJDomModelEtlIT {
 
+  @Test
+  public void addJDomDependencyAtIndex() throws IOException {
+    Dependency dependency = getSourceModel().getDependencies().get(0);
+    getDependenciesFromModel().add(1, dependency);
+    assertTransformation();
+  }
```

Comment:
Please add another test including comments. See review comment above

Reply:
Don't know if this exact feature needs such a test. Be free to add any test You consider useful.